CHANGELOG.md

```diff
@@ -13,13 +13,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.18] - 2024-02-15
+
+### Fixed
+
+-   Fixed outdated Flutter commands
```

Comment:
can you move these changes under the Unreleased section?

We will take care of adding the 0.10.18 section to the changelog 👍

Reply:
Can do!